### PR TITLE
HMA-3872 improve components release documentation

### PR DESCRIPTION
**Pull request by JamieCruwys**
# 📝 Description

https://jira.tools.tax.service.gov.uk/browse/HMA-3872

Updated readme to:
* Document release process in detail
* Explain the different ways of setting environment variables
* Add badge for current release version

# 🛠 Testing Notes
No code changes have been made

# 📸 Screenshots

### CODE REVIEW

**Review by chrisob55 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -716,6 +716,85 @@ The test library includes custom `ViewMatchers` and `ViewActions` for easier tes
...
+
+As part of the release process, [GitHub's CLI tool](https://github.com/cli/cli) is used. To have the entire process work smoothly you should have it set up on your machine.
+
+Please follow the [installation instructions](https://github.com/cli/cli#installation) to get it set up.
+
+[GitHub's CLI tool](https://github.com/cli/cli) is used to automatically create a pull request as part of the release process.
+
+**If you do not have GitHub's CLI set up on your machine, you will need to create the PR manually**
+
+### Running
```
Comment (jemmaSlater) [inline README.md:792]: Should this title be 'Releasing' and the earlier one be something else like 'Building' or 'Running'?

File: README.md [hunk 2]
```diff
@@ -716,6 +716,85 @@ The test library includes custom `ViewMatchers` and `ViewActions` for easier tes
 
 ## Releasing
 
-In order to release the library, you need to provide a Github personal access token with `repo` scope via a `GITHUB_TOKEN` environment variable - you can create a new token on [Github](https://github.com/settings/tokens/new).
+### Setup - Github personal access token
 
-To release the library, run `bundle exec fastlane tag_release` and choose to bump either the major, minor or patch when prompted. This will create a new tag, update the changelog and trigger the `publish` workflow on Bitrise.
+In order to release the library, you need to provide a Github personal access token with the following scope:
```
Comment (jemmaSlater) [inline README.md:721]: I needed to do this step just to build, so I think changing the wording away from 'in order to release' might be clearer as it is needed for the earlier steps too..

Comment (JamieCruwys) [reply README.md]: Good point - I've updated it in eb36e3f8623c7bc13282dd21ba5752465cb3745f

File: README.md (hunk 1 above)
Comment (JamieCruwys) [reply README.md]: Fixed in eb36e3f8623c7bc13282dd21ba5752465cb3745f

**Review by jemmaSlater — APPROVED** [review]

**Review by chrisob55 — APPROVED** [review]

**Pull request merged**